mne/preprocessing/stim.py

```diff
@@ -67,3 +91,92 @@ def eliminate_stim_artifact(raw, events, event_id, tmin=-0.005,
         elif mode == 'window':
             raw[picks, first_samp:last_samp] = data * window[np.newaxis, :]
     return raw
+
+
+def fix_stim_artifact(inst, events=None, event_id=None, tmin=0.,
+                      tmax=0.01, mode='linear', copy=False):
+    """Eliminates stimulations artifacts from instance
+
+    The instance will be modified in place (no copy)
+
+    Parameters
+    ----------
+    inst : instance of evoked or epochs
+        instance
+    events : array, shape (n_events, 3)
+        The list of events. No need when inst is epochs or evoked
```

Comment:
Say: "Required only when inst is raw" instead of "No need when inst is epochs or evoked"